packages/components/src/components/SquiggleViewer/ExpressionViewer.tsx

```diff
@@ -38,18 +47,16 @@ export const getBoxProps = (
   switch (value.tag) {
     case "Number":
       return {
-        preview: <NumberShower precision={3} number={value.value} />,
+        preview: <NumberShower precision={4} number={value.value} />,
         children: () => (
           <div className={clsx("font-semibold text-indigo-800", leftMargin)}>
-            <NumberShower precision={3} number={value.value} />
+            <NumberShower precision={4} number={value.value} />
           </div>
         ),
       };
     case "Dist": {
-      const distType = value.value.tag;
-
       return {
-        heading: `${distType} Distribution`,
+        preview: <DistPreview dist={value.value} environment={environment} />,
```

Comment:
Hmm, so we don't show the dist type (symbolic/sampleset/pointset) anymore? That solves the problem with long `Lognormal(...)` strings, but I'm surprised you've decided to remove it, it seemed useful.

Maybe we could keep the heading, or show it on hover on preview?

Reply:
I think we should add it when it drops down, later. I also like it as information, but it didn't fit the header.